botorch/optim/optimize_mixed.py

```diff
@@ -551,7 +663,8 @@ def continuous_step(
 def optimize_acqf_mixed_alternating(
```

Comment:
Let's update the docstring of the function to reflect the broader support, this currently says
> Optimizes acquisition function over mixed binary and continuous input spaces.